mne/viz/topomap.py

```diff
@@ -2001,15 +1949,34 @@ def plot_psds_topomap(
     import matplotlib.pyplot as plt
     sphere = _check_sphere(sphere)
 
+    if cbar_fmt == 'auto':
+        cbar_fmt = '%0.1f' if dB else '%0.3f'
+
+    if vmin is not None or vmax is not None:
+        if vlim == (None, None):
+            msg = ('Since you didn\'t specify "vlim", your provided values of '
+                   '"vmin" and "vmax" will be used.')
+            vlim = (vmin, vmax)
+        else:
+            msg = ('Your provided values for "vlim" will be used, and "vmin" '
+                   'and "vmax" will be ignored.')
+        warn('"vmin" and "vmax" are deprecated in favor of a single argument '
+             '"vlim". ' + msg, DeprecationWarning)
```

Comment:
"and will be removed in 0.21" somewhere in there. Usually we just say "vmin and vmax are deprecated and will be removed in 0.21, use vlim instead"